spec/requests/users_spec.rb

```diff
@@ -1,7 +1,39 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
+RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before do
+      get user_posts_path(user_id: 1)
+    end
+
+    it 'returns correct status' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template('posts/index')
```

Comment:
- [ ] This test should expect to render the right template for the users/index action. Kindly fix it.